### PR TITLE
Add folder & file-unknown icons

### PR DESCRIPTION
**Pull request by Neet-Nestor**
Add 2 new icons
- folder
- file-unknown

### CODE REVIEW

File: packages/gestalt/src/icons/file-unknown.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg"><g fill="#8e8e8e"><path d="M18.5 20.5h-13v-17H14v3A1.5 1.5 0 0 0 15.5 8h3zm1.914-14.086l-4.828-4.828A2 2 0 0 0 14.172 1H5.5A2.5 2.5 0 0 0 3 3.5v17A2.5 2.5 0 0 0 5.5 23h13a2.5 2.5 0 0 0 2.5-2.5V7.828a2 2 0 0 0-.586-1.414z"/><path d="M11.754 8.37C10.235 8.37 9 9.578 9 11.06a.754.754 0 1 0 1.508 0c0-.65.56-1.18 1.246-1.18S13 10.41 13 11.06c0 .397-.23.802-.559.985-.835.465-1.44 1.55-1.44 2.58a.754.754 0 1 0 1.507 0c0-.524.351-1.086.665-1.261.811-.45 1.335-1.355 1.335-2.304 0-1.482-1.235-2.688-2.754-2.688M11.754 16.25a.875.875 0 1 0 .001 1.75.875.875 0 0 0 0-1.75"/></g></svg>
```
Comment (pfarejowicz) [inline packages/gestalt/src/icons/file-unknown.svg]: This one doesnt look right, it has 2 `path` elements, are these from PDS?

Comment (Neet-Nestor) [reply packages/gestalt/src/icons/file-unknown.svg]: Yes. I download from it by myself and compared and it just has 2 paths.

Comment (Neet-Nestor) [reply packages/gestalt/src/icons/file-unknown.svg]: I have contacted Vickie and update this icon! :)

File: packages/gestalt/src/icons/file-unknown.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="m11.754 16.25c-.483 0-.875.392-.8740019.876 0 .483.3920019.874.8750019.874s.875-.392.875-.875-.392-.875-.875-.875zm0-7.88c-1.519 0-2.754 1.208-2.754 2.69 0 .416.338.754.754.754s.754-.338.754-.754c0-.65.56-1.18 1.246-1.18s1.246.53 1.246 1.18c0 .397-.23.802-.559.985-.835.465-1.44 1.55-1.44 2.58-.015.416.31.766.726.781s.766-.31.781-.726c.001-.018.001-.037 0-.055 0-.524.351-1.086.665-1.261.811-.45 1.335-1.355 1.335-2.304 0-1.482-1.235-2.688-2.754-2.688zm8.66-1.956-4.828-4.828c-.375-.375-.884-.586-1.414-.586h-8.672c-1.381 0-2.5 1.119-2.5 2.5v17c0 1.381 1.119 2.5 2.5 2.5h13c1.381 0 2.5-1.119 2.5-2.5v-12.672c0-.53-.211-1.039-.586-1.414zm-1.914 14.086h-13v-17h8.5v3c0 .828.672 1.5 1.5 1.5h3z" fill="#8e8e8e" fill-rule="evenodd"/></svg>
```
Comment (pfarejowicz) [inline packages/gestalt/src/icons/file-unknown.svg]: Can you check again with the PDS team? This SVG is not optimized, a simple pass through SVGO yields a 26% reduction in size
image

**Comment by Neet-Nestor** [comment]
Updated icon!

**Pull request merged**